lib/swaps/Swaps.ts

```diff
@@ -104,6 +106,60 @@ class Swaps extends EventEmitter {
     this.pool.on('packet.swapError', this.handleSwapError);
   }
 
+  /**
+   * verify that the order still exists on orderBook
+   */
+  private isOrderOnBook = (pairId: string, orderId: string): boolean => {
+    this.logger.debug('checking order book for ' + pairId + ':' + orderId);
+    return true;
+  }
+
+  /**
+   * verify that the order still exists on orderBook
+   */
+  private checkAndHoldAmount = (deal: SwapDeal): number => {
+    this.logger.debug('checking and holding amount for swap ' + deal.pairId + ':' + deal.orderId + ':' + deal.takerAmount);
+    return deal.takerAmount;
+  }
+
+  /**
+   * Send an error to peer. set reqId if packet is a response to a request.
+   */
+  private sendErrorToPeer = (peer: Peer, r_hash: string, errorMessage: string, reqId?: string) => {
+    const errorBody: packets.SwapErrorPacketBody = {
+      r_hash,
+      errorMessage,
+    };
+    this.logger.debug('Sending swap error to peer: ' + JSON.stringify(errorBody));
+    peer.sendPacket(new packets.SwapErrorPacket(errorBody, reqId));
+    return;
+  }
+
+  /**
+   * verify LND setup. Make sure we are connected to BTC and LTC and that
+   * the peer is also connected to these networks.
+   */
+  private verifyLNDSetup = (deal: SwapDeal, peer: Peer) => {
+    if (!peer.getLndPubKey(deal.takerCurrency)) {
+      return 'peer did not provide an LND PubKey for ' + deal.takerCurrency;
+    }
+
+    if (!peer.getLndPubKey(deal.makerCurrency)) {
+      return 'peer did not provide an LND PubKey for ' + deal.makerCurrency;
```

Comment:
The log messages in this method are redundant. Could you put them into a method?

Reply:
Don't see real value in making it a method now. will leave it like this.